### ISSUE TITLE
[RuntimeTest][iOS] Application crash with `mini-generic-sharing.c:899, condition `oti' not met`

### ISSUE THREAD

**Issue by jeromelaban** [unoplatform/uno#8167]
### Current behavior

Runtime tests may fail randomly with the following message:

```
2022-02-21 21:09:08.845540+0000  localhost SamplesApp[3799]: error: * Assertion at /Users/builder/jenkins/workspace/archive-mono/2020-02/ios/release/mono/mini/mini-generic-sharing.c:899, condition `oti' not met
```

### Expected behavior

The app does not crash

### How to reproduce it (as minimally and precisely as possible)

Run the runtime tests using an iOS simulator

### Workaround

None.

### Affected platforms

iOS

### Anything else we need to know?

.NET 6 may have fixed this issue.

**Comment by jeromelaban** [comment]
This has been fixed since then by the .NET runtime.

**Comment by jeromelaban** [comment]
This issue is not fixed, and has been found in .NET 7 runtime tests

**Comment by MartinZikmund** [comment]
@jeromelaban is this still happening?

**Comment by jeromelaban** [comment]
Yes it's still happening in net9, but it might be fixed in net10.

**Comment by MartinZikmund** [comment]
# Repro generator

This issue was classified as not reproducible with a code sample or test.

**Reason:** iOS-specific runtime test crash — cannot reproduce on Skia.
